conda_smithy/configure_feedstock.py

```diff
@@ -367,14 +367,14 @@ def _render_ci_provider(provider_name, jinja_env, forge_config, forge_dir, platf
 
 
 def _circle_specific_setup(jinja_env, forge_config, forge_dir):
-    # If the recipe supplies its own conda-forge-build-setup script,
+    # If the recipe supplies its own run_conda_forge_build_setup script,
     # we use it instead of the global one.
     cfbs_fpath = os.path.join(forge_dir, 'recipe', 'run_conda_forge_build_setup_linux')
 
     build_setup = ""
     if os.path.exists(cfbs_fpath):
         build_setup += textwrap.dedent("""\
-            # Overriding global conda-forge-build-setup with local copy.
+            # Overriding global run_conda_forge_build_setup with local copy.
             source /home/conda/recipe_root/run_conda_forge_build_setup_linux
```

Comment:
This should be `run_conda_forge_build_setup_linux`.

Reply:
Though maybe in cases like these we don't need to be OS specific. Leave it up to you.